### PR TITLE
SEO-176062-Windowsforms-duplicate-Ug-titles

### PR DESCRIPTION
**Pull request by Edithabich**
HI @MallikaRK ,
Have updated UG titles please review.
Thanks.

Regards,
Edith Abich.

### CODE REVIEW

File: WindowsForms/Classic/ComboBox/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms ComboBoxAdv control | Syncfusion
+title: Overview of Windows Forms ComboBoxAdv control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Classic/ComboBox/Overview.md:3]: Overview of Windows Forms ComboBox Control | Syncfusion

File: WindowsForms/Classic/Currency-Edit/Overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms ContextMenuStrip control | Syncfusion
+title: Overview of Windows Forms CurrencyEdit control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Classic/Currency-Edit/Overview.md:3]: Control, the first letter should be in caps

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Classic/Currency-Edit/Overview.md (hunk 2 above)
Comment (Edithabich) [reply WindowsForms/Classic/Currency-Edit/Overview.md:3]: resolved

File: WindowsForms/Classic/ComboBox/Overview.md (hunk 1 above)
Comment (Edithabich) [reply WindowsForms/Classic/ComboBox/Overview.md:3]: resolved

File: WindowsForms/Classic/Currency-Edit/Overview.md (hunk 2 above)
Comment (Edithabich) [reply WindowsForms/Classic/Currency-Edit/Overview.md:3]: resolved

File: WindowsForms/layoutmanagers/CardLayout/Overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms LayoutManagers control | Syncfusion
+title: Overview of Windows Forms CardLayout Control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/layoutmanagers/CardLayout/Overview.md:3]: Use LayoutManagers

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Pull request closed without merging**